example/src/main/res/values-de/strings.xml

```diff
@@ -0,0 +1,26 @@
+<resources>
+    <!-- Main Strings -->
+    <string name="app_name">AppIntro Beispiel</string>
+    <string name="drawer_header">Drawer Header</string>
+    <string name="launch_demo">Demo starten</string>
+
+    <!-- 1st Tab Group -->
+    <string name="default_intro">Dies is die einfachste Art, die AppIntro Bibliothek zu benutzen. Nur wenig code ist nötig, um dies umzusetzen und es gibt zwei Layoutoptionen. Drücke den Button unten, um die Demo zu starten.</string>
```

Comment:
Wrong case in `Nur wenig code ist nötig` -> `Nur wenig Code ist nötig`.
Wrong grammar in `um dies einzusetzen` -> `um diese einzusetzen`.